### DISCUSSION TITLE
Set env var at niri start

### DISCUSSION THREAD

**Discussion by Buer-Nahida** [YaLTeR/niri#214] [Q&A] [answered]
I want set some env var at niri start, such as `GDK_BACKEND=wayland` `QT_QPA_PLATFORM=wayland` `QT_QPA_PLATFORMTHEME=qt5ct` and more, I'm also read wiki and comments but I'm not found answers, How can I do it?

**Comment by YaLTeR** [comment, marked as answer]
Hey, so niri-session runs through your shell, so you can set them in `~/.bashrc` or a similar file for your shell.

**Reply by Buer-Nahida** [reply]
Yes, put there env variables in `/etc/environment` and reboot my computer. in next open, There are working in niri.
